.flake8

```diff
@@ -1,3 +0,0 @@
-[flake8]
-max-line-length = 100
-extend-ignore = E203
```

Comment:
is this OK to not have a line ignoring this in pyproject.toml

Reply:
Yes. We moved to ruff + black (or ruff + ruff format, I don't recall), so the line length is not relevant anymore.